examples/buttons/index.html

```diff
@@ -0,0 +1,108 @@
+    <!-- The above 3 meta tags *must* come first in the head; any other head content must come *after* these tags -->
+    <title>Buttons Example</title>
+
+    <!-- build:template
+    <script src="../../<%= rockyjs_path %>"></script>
+    /build -->
+    <!-- build:remove -->
+    <!-- this template/remove construct is a workaround as processhtml doesn't support variables for values -->
+    <script src="../../src/html-binding.js"></script>
+    <script src="../../src/symbols-manual.js"></script>
+    <script src="../../src/symbols-generated.js"></script>
+    <script src="../../src/transpiled.js"></script>
+    <!-- /build -->
+
+    <script src="modules/buttons.js"></script>
+
+    <!-- build:css ../../css/bootstrap.min.css -->
+    <link href="../../html/css/bootstrap.min.css" rel="stylesheet">
+    <!-- /build -->
+    <!-- build:css ../../css/style.css -->
+    <link href="../../html/css/style.css" rel="stylesheet">
+    <!-- /build -->
+
+    <!-- HTML5 shim and Respond.js for IE8 support of HTML5 elements and media queries -->
+    <!-- WARNING: Respond.js doesn't work if you view the page via file:// -->
+    <!--[if lt IE 9]>
+    <script src="https://oss.maxcdn.com/html5shiv/3.7.2/html5shiv.min.js"></script>
+    <script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
+    <![endif]-->
+
+    <link rel="stylesheet" href="../common/css/style.css">
+</head>
+<body>
+<div class="container">
+    <div class="page-header">
+        <h1>Buttons Example</h1>
+        <p class="lead">
+            This is a simple example of how to implement Pebble button clicks in RockyJS using a keyboard's directional arrow keys. 
+        </p>
+    </div>
+    
+    <div class="col-md-6">
+        <canvas id="pebble" class="rocky" width="432" height="504"></canvas>    
+    </div>
+
+    <div class="col-md-6">
+        <h2>What's going on?</h2>
+        <p>
+            This example demonstrates how to interact with RockyJS bound to a canvas. Using the left, right, up, and down arrows on your keyboard you can simulate the back, select, up, and down buttons of the Pebble respectively. 
+        </p>
+
+        <code>onPress(type, callback)</code>
+        <p>
+            Options for <code>type</code> are 'back', 'select', 'up', or 'down'.
+        </p>
+        <p>
+            Source for this example can be found at [link TBD].
+    </div>
+    <!-- build:template
+    <%= github_banner %>
+    /build -->
+</div>
+</body>
+<script>
+var rocky = Rocky.bindCanvas(document.getElementById("pebble"));
+rocky.export_global_c_symbols();
+
+/**** App ****/
+
+var currentText = "Press an arrow key";
+
+rocky.update_proc = function (ctx, bounds) {  
+  graphics_context_set_text_color(ctx, GColorBlack);
+  graphics_draw_text(ctx, currentText, fonts_get_system_font(FONT_KEY_GOTHIC_24_BOLD), 
+                     bounds, GTextOverflowModeWordWrap, GTextAlignmentLeft, null);
+};
+
+(function() {
+  onPress("back", function() {
```

Comment:
you put this logic into an anonymous function to not pollute the global namespace but `currentText` as well as `rocky` are still globals.